xmtp/src/conversation.rs

```diff
@@ -190,9 +190,6 @@ where
     }
 
     pub async fn initialize(&self) -> Result<(), ConversationError> {
-        self.client
```

Comment:
Don't we want to keep this one so we can send invites to the most up-to-date set of participant installations?

Reply:
I wonder if it makes sense to separate out refreshing installations (i.e. pre-fetching contacts from the network whenever is optimal), and creating sessions with those installations (which ideally happens only on send).

Reply:
But currently we're fetching on every send regardless (refresh interval of 0), and a low interval might actually be ideal, so maybe it matters less

Reply:
Agree - the invite is effectively a message, ideally we can use similar code-paths for invites compared to messages

Reply:
Thats a fair point. I got a little over zealous in this case. will update